### ISSUE TITLE
Node 15 Support

### ISSUE THREAD

**Issue by nschonni** [sass/node-sass#2965]
Version 15 is supposed to be released on October 20th, 2020. Few things that should probably happen to make it smoother going forward:

- [ ] Promote the fixups in the v5 branch
- [x] Drop testing on unsupported Node versions (<10)
- [ ] Allow people to try and build on old node version, but show warning that we won't support issues opened for those platforms
- [ ] Hold off 3.6 libsass upgrade for separate major release once there is a major version people can pin to

**Comment by nschonni** [comment]
@xzyfer I believe the release is expected out today (the 20th). It may just be a matter of restarting the CIs on https://github.com/sass/node-sass/pull/2983 or we might have to wait for AppVeyor for the x86 build

**Comment by xzyfer** [comment]
Traditionally we have been blocked on appveyor and Travis to add the new node versions.

**Comment by xzyfer** [comment]
I can build the OSX binaries which will mean we're not blocked on Travis.

**Comment by nschonni** [comment]
I think OSX/Linux should be fine on the GitHub Actions, since they should upload the binaries. I think the gaps will be:
- [ ] Windows x86 on AppVeyor
- [ ] Apline on GitHub Actions will require a new nodejs/docker-node PR and publish on official-images repo
- [ ] FreeBSD binraries from @saper unless we get the Cirrus CI figured out

Remembered that we're actually fully of Travis now, so we're not waiting for that part anymore

**Comment by xzyfer** [comment]
I'm not sure I understand the alpine issue. Alpine images are usually
released alongside the Debian images. Are they not suitable?

**Comment by nschonni** [comment]
We use the Node.js Alpine images from https://github.com/nodejs/docker-node, which will need to release a v15 version. Once a PR is merged there, it creates another PR to https://github.com/docker-library/official-images, which is then merged and built out on their CI to publish to the Docker Hub

**Comment by xzyfer** [comment]
Sorry I think I'm just out of touch with how the new GitHub checks work.
I'll read up on it.

**Comment by nschonni** [comment]
The Alpine stuff is here https://github.com/sass/node-sass/blob/master/.github/workflows/alpine.yml, it pulls and runs on the "official image" for Node.js. Rest of the Actions use the https://github.com/actions/setup-node/ to install a particular version on Node.js

**Comment by xzyfer** [comment]
I quickly skimmed the action and it looks like it might be using an arch
Linux mode binary. I may still be missing something however

Typically we build the binaries on CentOS for better compatibility across
Linux distributions. Just something we should keep note of incase we start
seeing reports of issues with the binaries.

**Comment by nschonni** [comment]
@xzyfer so funny story, looks like AppVeyor is actually OK, but all platforms are failing to read the built binary and failing on the test step 😕 

PS: PR for the Alpine docker images is https://github.com/nodejs/docker-node/pull/1372

**Comment by wiesson** [comment]
I just wanted to mention that yarn did rebuild node-sass for me without any problems (needed to delete `node_modules` though)

**Comment by dhulme** [comment]
My build server's docker image (alpine) upgraded to Node 15 and the builds broke because binaries aren't available yet. I've fixed by changing the image to use LTS but would be great to get these binaries out 👍 Thanks!

**Comment by jledgett** [comment]
@nschonni Noticed you closed my apparent duplicate issue.   Looking through this thread, I don't see an ETA though.  Thanks in advance.

**Comment by robrich** [comment]
Will there be a node-sass@4.x build for Node 15?

**Comment by nschonni** [comment]
No, it will only be supported in 5 https://github.com/sass/node-sass#node-version-support-policy

**Comment by herbievine** [comment]
Is there a v5.x?

**Comment by nschonni** [comment]
That bump will be handled as part of the NPM publish apparently https://github.com/sass/node-sass/pull/3012

**Comment by nschonni** [comment]
Closing as v5 is out with Node 15 support. Thanks @xzyfer !

**Comment by xzyfer** [comment]
@nschonni you da real mvp
